docs/pages/database-access/guides/sql-server-ad.mdx

```diff
@@ -0,0 +1,324 @@
+  Windows machine joined to your Active Directory domain:
+
+  ```code
+  $ setspn -L alice
+  ```
+
+  To assign an SPN to a user account, use the following command:
+
+  ```code
+  $ setspn -s user/alice alice
+  ```
+</Admonition>
+
+You can verify entries in the keytab file using `klist` command:
+
+```bash
+$ klist -ke teleport.keytab
+Keytab name: FILE:teleport.keytab
+KVNO Principal
+---- --------------------------------------------------------------------------
+   5 alice@DBADIR.TELEPORTDEMO.NET (aes256-cts-hmac-sha1-96)
+   2 bob@DBADIR.TELEPORTDEMO.NET (aes256-cts-hmac-sha1-96)
+```
+
+<Admonition type="warning">
+  You must update the keytab file after updating a user's password to avoid
+  authentication failures.
+</Admonition>
+
+## Step 5/7. Set up Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+<Admonition type="note">
+  Teleport Database Service must run on a Linux server joined to the same
+  Active Directory domain as the SQL Server.
+</Admonition>
+
+Start Teleport Database Service. Make sure to update `--auth-server` to point to
+your Teleport Proxy Service address and `--uri` to the SQL Server endpoint.
+
+  ```bash
+  $ teleport db start \
+    --token=/tmp/token \
+    --auth-server=teleport.example.com:3080 \
+    --name=sqlserver \
+    --protocol=sqlserver \
+    --uri=sqlserver.example.com:1433 \
+    --ad-keytab-file=/path/to/teleport.keytab \
+    --ad-domain=EXAMPLE.COM \
+    --ad-spn=MSSQLSvc/sqlserver.example.com:1433 \
+    --labels=env=dev
+  ```
+
+Provide Active Directory parameters:
+
+| Flag | Description |
+| ---- | ----------- |
+| `--ad-keytab-file` | Path to Kerberos keytab file generated above. |
+| `--ad-domain` | Active Directory domain (Kerberos realm) that SQL Server is joined. |
+| `--ad-spn` | Service Principal Name for SQL Server to fetch Kerberos tickets for. |
+
+<Admonition type="tip">
+  You can start Teleport Database Service using a configuration file instead of
+  CLI flags. See [YAML reference](../reference/configuration.mdx).
+</Admonition>
+
+### Service Principal Name
+
+You can use `adutil spn search` command to see the SPNs registered for your SQL
+Server. Typically, they take a form of `MSSQLSvc/<name>.<ad-domain>:<port>`.
+
+For example, an AWS RDS SQL Server named `sqlserver` and joined to an AWS managed
+Active Directory domain `EXAMPLE.COM` will have the following SPNs registered:
+
+```bash
+$ adutil spn search
+...
+CN=EC2AMAZ-4KN05DU,OU=RDS,OU=AWS Reserved,DC=example,DC=com:
+    MSSQLSvc/sqlserver.example.com:1433
+    MSSQLSvc/EC2AMAZ-4KN05DU.example.com:1433
+    MSSQLSvc/EC2AMAZ-4KN05DU.example.com
+...
+```
+
+Alternatively, you can look SPNs up in the Attribute Editor of the Active Directory
+Users and Computers dialog on your AD-joined Windows machine. The RDS SQL Server
+object typically resides under the AWS Reserved / RDS path:
+
+![SPN](../../../img/database-access/guides/sqlserver/spn@2x.png)
```

Comment:
The "Attribute Editor" tab is present only when "View>Advanced Features" is enabled.